content/docs/authorization/v2.0 Tech Preview/configuration/powermax/_index.md

```diff
@@ -0,0 +1,150 @@
+            - powermax-creds # credential secret for primary unisphere, e.g., powermax-creds
+    managementServers:
+        - url: "https://localhost:9400" # primary unisphere endpoint
+        arrayCredentialSecret: powermax-creds # credential secret, e.g., powermax-creds
+        skipCertificateValidation: true
+   ```
+
+6. Enable CSM Authorization in the driver installation applicable to your installation method.
+
+    **Operator**
+
+    Refer to the [Install Driver](../../../../deployment/csmoperator/drivers/powermax/#install-driver) section to edit the parameters in the Custom Resource to enable CSM Authorization.
+
+    Under `modules`, enable the module named `authorization`:
+
+    - Update the `enabled` field to `true.`
+
+    - Update the `image` to the image of the CSM Authorization sidecar. In most cases, you can leave the default value.
+
+    - Update the `PROXY_HOST` environment value to the hostname of the CSM Authorization Proxy Server. `csm-authorization.com` is a placeholder for the proxyHost. See the administrator of CSM for Authorization for the correct value.
+
+    - Update the `SKIP_CERTIFICATE_VALIDATION` environment value to `true` or `false` depending on if you want to disable or enable certificate validation of the CSM Authorization Proxy Server.
+
+    - Do not update the `configVersion`. You will notice in the example that it is set to v1.11.0, this ensures that checks on version support do not prevent deployment of the v2.0.0-alpha authorization tech preview.
+
+    Example: 
+
+    ```yaml
+    modules:
+      # CSI Powermax Reverseproxy is a mandatory module for Powermax
+      - name: csireverseproxy
+        # enabled: Always set to true
+        enabled: true
+        forceRemoveModule: true
+        configVersion: v2.10.0
+        components:
+        - name: csipowermax-reverseproxy
+        # image: Define the container images used for the reverse proxy
+        # Default value: None
+          image: dellemc/csipowermax-reverseproxy:v2.10.0
+          envs:
+          # "tlsSecret" defines the TLS secret that is created with certificate
+          # and its associated key
+          # Default value: None
+          # Example: "tls-secret"
+          - name: X_CSI_REVPROXY_TLS_SECRET
+              value: "csirevproxy-tls-secret"
+          - name: X_CSI_REVPROXY_PORT
+              value: "2222"
+          - name: X_CSI_CONFIG_MAP_NAME
+              value: "powermax-reverseproxy-config"
+          # deployAsSidecar defines the way reversproxy is installed with the driver
+          # set it true, if csm-auth is enabled / you want it as a sidecar container
+          # set it false, if you want it as a deployment
+          - name: "DeployAsSidecar"
+              value: "true"
+
+      # Authorization: enable csm-authorization for RBAC
+      - name: authorization
+        # enable: Enable/Disable csm-authorization
+        enabled: true
+        configVersion: v1.11.0
```

Comment:
v1.12.0?

Reply:
Updated. Before release, we will have to verify that the versions align either way.